andela/mrm_api
# #161179604 Implement events check in functionality

#### What does this PR do?
- Saves the event Id and room for the event that a user has checked in to

#### Description of Task to be completed?
- Create a model for saving the events data
- Create a mutation to allow check in data to be saved
- Add tests for the check in feature

#### How should this be manually tested?
- Run the server
- Run the `eventCheckin` mutation with `calendarId` and `eventId` as the parameters
- It should return the details for that specific event check in
- Screenshots are attached below

#### What are the relevant pivotal tracker stories?
[#161179604](https://www.pivotaltracker.com/story/show/161179604)

#### Screenshots
_Correct Check In Format_
<img width="1027" alt="screen shot 2018-10-18 at 10 33 17" src="https://user-images.githubusercontent.com/33119403/47138584-7f896800-d2c2-11e8-847b-ad389fb35146.png">


_Check In with the Same Details_
<img width="1014" alt="screen shot 2018-10-18 at 10 35 27" src="https://user-images.githubusercontent.com/33119403/47138585-7f896800-d2c2-11e8-9d58-65343f1c1244.png">


_Check In with Unregistered Calendar Id_
<img width="1091" alt="screen shot 2018-10-18 at 10 36 18" src="https://user-images.githubusercontent.com/33119403/47138586-7f896800-d2c2-11e8-8d6e-e701eeac4013.png">
